src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -14,7 +14,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
+    <UsePublicApiAnalyzers>false</UsePublicApiAnalyzers>
+    <EnablePackageValidation>false</EnablePackageValidation>
```

Comment:
@joperezr - should we only EnablePackageValidation on `IsPackable=true` projects?

Reply:
Yup, that's what we are doing, I think this is needed because the dashboard project is actually packable, but I'll double check.

Reply:
It shouldn't be packable. We haven't shipped it since preview2 - https://www.nuget.org/packages/Aspire.Dashboard/.  (we should probably deprecate that package).

It is a standalone web app, so it doesn't make sense to make a nupkg out of it.